measure_extinction/extdata.py

```diff
@@ -551,10 +551,16 @@ def trans_elv_alav(self, av=None, akav=0.112):
             if av is None:
                 if "AV" not in self.columns.keys():
                     self.calc_AV(akav=akav)
-                av = _get_column_val(self.columns["AV"])
+            fullav = self.columns["AV"]
+            if len(np.atleast_1d(fullav)) == 1:
+                fullav = np.array([fullav, 0.0])
             for curname in self.exts.keys():
-                self.exts[curname] = (self.exts[curname] / av) + 1
-                self.uncs[curname] /= av
+                tuncs = np.sqrt(
+                    np.square(self.uncs[curname] / self.exts[curname])
+                    + np.square(fullav[1] / fullav[0])
+                )
+                self.exts[curname] = (self.exts[curname] / fullav[0]) + 1
+                self.uncs[curname] = tuncs * self.exts[curname]
```

Comment:
When I do the math of error propagation, I find that you should be multiplying by E(lambda-V)/A(V), which is actually self.exts[curname]-1. Please correct me if I am wrong. I am using
<img width="413" alt="Screen Shot 2021-11-29 at 2 42 39 PM" src="https://user-images.githubusercontent.com/45097951/143932135-b5b743cc-9b97-48be-9878-a508f1da43a7.png">
with x=E(lambda-V) and y=A(V)...

Reply:
I'm not familiar with this formula.  

I'm was following the simple error propagation where if multiplying/dividing two numbers then:

sigma_f^2 / f^2 = sigma_a^2 / a^2 + sigma_b^2 / b^2

where a = E(lambda-V) and b = A(V) and

f = a/b + 1

Use the look inside button at https://www.amazon.com/Introduction-Error-Analysis-Uncertainties-Measurements/dp/093570275X

The book may be in my office, but might still be in a box and not on my selves yet.

Reply:
And I see your equation is right after the one I'm using.  Hah.

Reply:
If you use f=a/b in that "general" equation with the derivatives, you get your equation. However, there is a +1 here that makes a difference. The general equation is on page 79 (found the book in your office).

Reply:
So, is the only change to change to `self.uncs[curname] = tuncs * (self.exts[curname] - 1.0)`?

Reply:
Correct!

Reply:
Ok.  This can make for negative uncertainties, so will use np.absolute make sure it is positive.

Reply:
I see. Probably the better way to do it then, is to calculate tuncs immediately correctly, so there is no need to multiply at the end.

Reply:
Will make larger uncs for the small values of A(lambda)/A(V).  Interesting.